code/lib/core-events/src/errors/server-errors.ts

```diff
@@ -44,3 +44,78 @@ export class NxProjectDetectedError extends StorybookError {
     `;
   }
 }
+
+export class MissingFrameworkFieldError extends StorybookError {
+  readonly category = Category.CORE_COMMON;
```

Comment:
I'm not 100% sure about the category and its purpose. What if the same error needs to be thrown in 2 different packages? What will be the category then?

Reply:
Yes that's a challenge to think about, but currently the error originates from a validation function in core-common, which can (and is) be used as part of different packages, but it always comes from core-common, hence the category. If you have another idea I'd love to hear it though!

Reply:
My feeling is that maybe we don't need the category?
We can add contextual information to telemetry when we catch it, and doesn't the stack trace show which package/function the error originated from?

Reply:
We spoke on Tuple: we'll be experimenting and getting feedback over time